src/main/java/org/broadinstitute/hellbender/engine/ProgressMeter.java

```diff
@@ -210,15 +230,17 @@ public void stop() {
         Utils.validate( !stopped, "the progress meter has been stopped already");
         this.stopped = true;
         currentTimeMs = timeFunction.getAsLong();
-        logger.info(String.format("Traversal complete. Processed %d total records in %.1f minutes.", numRecordsProcessed, elapsedTimeInMinutes()));
+        logger.info(String.format("Traversal complete. Processed %d total %s in %.1f minutes.", numRecordsProcessed, recordLabel, elapsedTimeInMinutes()));
```

Comment:
Maybe now that you are here it may be useful to add a more verbose output by:;

1. Call `printProgress()` before logging the last line to see the last locus processed, and total processing rate.
2. It says "Traversal complete", but it may be used in other contexts. Maybe rephrase the message to "Processing complete: %s total %s in %.1f minute" may be useful for other use cases (and the "Traversal complete" could be moved to the `GATKTool` specific method...
3. Similarly to the progress logger in HTSJDK, it may be useful to customize "Processed" with tool-specific names ("Sorted", "Merged", "Traversed"...). This is just something that may be useful, but I really do not need it.

I'm very interested in have 1 to "profile" tools easier just by looking at the last row of the progress meter... but these are just suggestions...